CMakeLists.txt

```diff
@@ -120,7 +120,12 @@ if(DJINNI_WITH_JNI)
 
   # Do not use the host's jni.h on Android as it is provided automatically by the NDK
   if(NOT ANDROID)
+    # avoid error messages if awt is not found, what seems to be on some sytems sometimes the case
+    set(JAVA_AWT_LIBRARY NotNeeded)
+    set(JAVA_AWT_INCLUDE_PATH NotNeeded)
     find_package(JNI REQUIRED QUIET)
+    # since we are lazy using all JNI_INCLUDE_DIRS, instead of listing that we use, remove NotNeeded from all:-)
+    list(REMOVE_ITEM JNI_INCLUDE_DIRS NotNeeded)
```

Comment:
Just out of curiosity: What would the not-so-lazy approach look like? 🤔

Reply:
using the parts JNI_INCLUDE_DIRS are composed from explicit.
Most likely that are JAVA_INCLUDE_PATH and JAVA_INCLUDE_PATH2, but I would have to look it up if there are not others

Reply:
Do you think it would be worth to give that a try? If you cannot get it to work we can still stick with this fix for the lazy approach. :P

Reply:
Well, it works, if AWT is not there, find JNI will not raise an error.
I have just to clean the NotNeeded message to cmake after reaching the goal, 
this fully fixes #12

Reply:
but lets see ...